### ISSUE TITLE
Not getting perfect weights

### ISSUE THREAD

**Issue by preritk** [preritk/XOR_NEURAL#1]
I'm unable to get correct weights . Please help me out in the backpropogation and how to optimise the code.

**Comment by ghost** [comment]
The learning rate is 0.1 initially. You should start changing that. First increase it significantly.. like .6 or more and see the effect it has on the result.
